docs/platforms/javascript/common/configuration/integrations/openfeature.mdx

```diff
@@ -29,7 +29,8 @@ This integration only works inside a browser environment. It is only available f
 
 </Alert>
 
-The [OpenFeature](https://openfeature.dev/) integration tracks feature flag evaluations produced by the OpenFeature SDK. These evaluations are held in memory, and in the event an error occurs, sent to Sentry for review and analysis. **At the moment, we only support boolean flag evaluations.**
+The [OpenFeature](https://openfeature.dev/) integration tracks feature flag evaluations produced by the OpenFeature SDK. These evaluations are held in memory, and in the event an error occurs, sent to Sentry for review and analysis. **At the moment, we only support boolean flag evaluations.** This integration is available in
+Sentry SDK **versions 8.43.0 or higher.**
```

Comment:
Plural 'versions'?

Reply:
thought it flowed a little better, but I'm ok either way